README.md

```diff
@@ -14,9 +14,13 @@ Swift is architecture specific, so running the build command on macOS will creat
 
 To build on Linux and create a Linux binary, use Docker. For example:
 
-`$ docker run -v $PWD:/code -w /code swift:5.1 swift build`
+`$ docker run -v "$PWD:/code" -w /code swift:5.3 swift build`
```

Comment:
should we just change this to `swift:latest`?

Reply:
@tomerd doesn't exist

```
$ docker run -it --rm swift:latest swift -version
Unable to find image 'swift:latest' locally
docker: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable.
See 'docker run --help'.
$ docker run -it --rm swift swift -version
Unable to find image 'swift:latest' locally
docker: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable.
See 'docker run --help'.
```

Reply:
works for me??